conf/pipeline/rnaseq/utd_sysbio.config

```diff
@@ -0,0 +1,19 @@
+params {
+  config_profile_description = 'University of Texas at Dallas HPC cluster profile provided by nf-core/configs'
```

Comment:
Would suggest specifying is rna-seq specific one but otherwise gtg

Reply:
I'll address this in another PR